typescript/cli/src/submit/submitCommand.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  PopulatedTransaction,
+  PopulatedTransactionSchema,
+} from '@hyperlane-xyz/sdk';
+import { assert } from '@hyperlane-xyz/utils';
+
+import { WriteCommandContext } from '../context/types.js';
+import { readYamlOrJson } from '../utils/files.js';
+
+import { getSubmitterBuilder } from './submit.js';
+
+export async function runSubmit({
+  context,
+  transactionsFilepath,
+}: {
+  context: WriteCommandContext;
+  transactionsFilepath: string;
+}) {
+  const { submissionStrategy, chainMetadata, multiProvider, isDryRun } =
+    context;
+
+  assert(
+    submissionStrategy,
+    `Submission strategy required to submit transactions.
+  Please create a submission strategy, e.g. ./strategy.yaml.`,
+  );
+
+  const chain = submissionStrategy.chain;
+  const protocol = chainMetadata[chain].protocol;
+  const submitterBuilder = await getSubmitterBuilder<typeof protocol>({
+    submitterMetadata: submissionStrategy.submitter,
+    transformersMetadata: submissionStrategy.transforms ?? [],
+    multiProvider,
+    chain: submissionStrategy.chain,
+    isDryRun,
+  });
+  const transactions = getTransactions(transactionsFilepath);
+  const transactionReceipts = await submitterBuilder.submit(...transactions);
+
+  if (transactionReceipts)
+    console.log('🧾 Transaction receipts:', transactionReceipts);
```

Comment:
Go through logger. I thought the linter would prevent this, maybe that rule isn't enabled in the CLI. If not we should enable it